Merge Conflict/Assets/Scripts/UiManager.cs

```diff
@@ -154,10 +173,12 @@ private void SwitchMenu(string requestedMenu)
                     OpenMenu(_mainmenu);
                     AudioManager.Instance.PlayOpenMenuSound();
                 }
+
                 else
                 {
                     _playfield.enabled = true;
```

Comment:
_playfield = enabled; is now called in CloseAllMenus() and doesnt need to be called here